README.md

```diff
@@ -145,45 +145,45 @@ which is automatically converted to a Julia type, you will have override this
 via `@pywith EXPR::PyObject ...`.
 
 If you are already familiar with Python, it perhaps is easier to use
-`py"..."` and `py"""..."""` which are equivalent to Python's
+``py`...``` and ````py```...``````` which are equivalent to Python's
```

Comment:
The latter doesn't seem to work either. Does anyone know how to do that with GitHub's Markdown syntax?

Reply:
Just putting a space seems to work:
```
` py```...``` `
```

Reply:
Ah thanks!

Reply:
That still seems to leave a trailing whitespace highlighted, but probably not that noticeable.

Reply:
Ah, wait. I misunderstood!

Reply:
Should be fixed now